### DISCUSSION TITLE
Deploy ssr-vue example in vercel ...

### DISCUSSION THREAD

**Discussion by amirhossein-fzl** [vitejs/vite-plugin-react#241] [Q&A]
How can I deploy my vite + ssr project in vercel? I proceeded in my project like the following example:

https://github.com/vitejs/vite-plugin-vue/tree/main/playground/ssr-vue

This is exactly the same example that vite also wrote on the SSR page: https://vitejs.dev/guide/ssr.html

Github address of my project: https://github.com/amirhossein-fzl/resume

**Comment by ArnaudBarre** [comment]
main.vitejs.dev now has better templates with create-vite-extra https://main.vitejs.dev/guide/ssr.html

As the warning on this page shows, this API doesn't have the goal to be plug and play and easy to deploy.

Because this is question is also about vue on the repo of the react plugin I will go ahead and close the discussion
